Sources/Extensions/UIKit/UIImageExtensions.swift

```diff
@@ -96,6 +96,69 @@ public extension UIImage {
 		return newImage
 	}
 
+    /// Creates a copy of the receiver rotated by the given angle.
+    ///
+    ///     // Rotate the image by 180°
+    ///     image.rotated(by: Measurement(value: 180, unit: .degrees))
+    ///
+    /// - Parameter angle: The angle measurement by which to rotate the image.
+    /// - Returns: A new image rotated by the given angle.
+    @available(iOS 10.0, *)
```

Comment:
@guykogus This availability check only makes it available to iOS 10.0+, but I think we can add it also to tvOS 10.0+, macOS 10.12+ and watchOS 3.0+ :))  Like `@available(OSX 10.12, iOS 10.0, watchOS 3.0, tvOS 10.0, *)` Since Measurement is available for them too.
What you think?

Reply:
Wow, I love CI 💯 , seems like the build is breaking on those other platforms because of those missing availability checks.
Now I think is a requirement 😆